contrib/wdisplays/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "wdisplays"
+pkgver = "1.1.1"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = [
+    "glib-devel",
+    "meson",
+    "pkgconf",
+    "python-scour",
+    "wayland-progs",
+]
+makedepends = [
+    "gtk+3-devel",
+    "libepoxy-devel",
+    "wayland-protocols",
+]
+pkgdesc = "GUI display configurator for wlroots compositors"
```

Comment:
"Configurator" sounds very awkward here:
```suggestion
pkgdesc = "GUI display configuration tool for wlroots compositors"
```